### DISCUSSION TITLE
How to remove Clear icon?

### DISCUSSION THREAD

**Discussion by Productivix** [wojtekmaj/react-datetimerange-picker#53] [Q&A] [answered]
hi, 
I want to avoid to have the icon : 
So, according to the doc , I tried without success : 
```
 clearIcon={"Clear"}
 clearIcon={null}
 clearIcon={Clear}
```
what is the exact syntax please ?

**Comment by wojtekmaj** [comment, marked as answer]
Hmm, works for me 🤔

https://codesandbox.io/s/react-datetimerange-picker-no-clear-icon-0625rm?file=/src/App.js

**Reply by Productivix** [reply]
but : I still have the icon : see 
![Capture](https://user-images.githubusercontent.com/6737060/162678545-4c1c37f6-510d-4efd-953d-49091c21eeb4.JPG)
:

**Reply by wojtekmaj** [reply]
That's calendarIcon.

**Reply by Productivix** [reply]
it is an icon for me (!) ;-)..... but which icon do you speak of before so ?

**Reply by wojtekmaj** [reply]
There are two icons. clearIcon and calendarIcon. Set them both to null and you'll have zero icons.
